rest/route/project_validate.go

```diff
@@ -74,6 +74,9 @@ func (v *validateProjectHandler) Run(ctx context.Context) gimlet.Responder {
 	}
 
 	projectRef, err := model.FindMergedProjectRef(ctx, v.input.ProjectID, "", false)
+	if err != nil {
```

Comment:
err is handled by validator.CheckProject so this is not needed.

Reply:
ah, updated and put a comment there!